support/k8s/k8s-workload-registrar/mode-crd/config/spire-agent.yaml

```diff
@@ -108,16 +108,9 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: spire-agent
-      initContainers:
```

Comment:
Without this the spire-agents will crash couple of times until server is up on a fresh install.

![image](https://user-images.githubusercontent.com/694733/205895153-a0bf8b00-849e-4049-ab72-abd978b7f0ab.png)

Reply:
We'll happily package wait-for-it and maintain images for this if it helps, like @developer-guy suggested above.

Reply:
❤️ thank you @developer-guy and @dlorenc .. we will take you up on that!

We discussed this change today and concluded that it will be best to avoid the use of `wait-for-it` in our test suites, however we still want to use it in all of our tutorials. I left a comment on an existing issue about it here https://github.com/spiffe/spire-tutorials/issues/90#issuecomment-1343280662

Thanks again!